modal/mount.py

```diff
@@ -279,8 +280,11 @@ def _new(entries: list[_MountEntry] = []) -> "_Mount":
         rep = f"Mount({entries})"
 
         async def mount_content_deduplication_key():
+            def normalized_selected_files():
```

Comment:
We're most likely getting rid of this way of deduplicating mounts soon. It won't be needed when we don't have auto mounts.

Instead I'll try to make something simpler that possibly auto-detects if the function's definition file itself is added to the image already and use the path of that instead of adding a separate entrypoint mount